teps/0052-nested-triggers.md

```diff
@@ -0,0 +1,202 @@
+---
+status: proposed
+title: Nested Triggers
+creation-date: '2021-02-10'
+last-updated: '2021-02-10'
+authors:
+- '@jmcshane'
+---
+
+# TEP-0052: Nested Triggers
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [Use Cases (optional)](#use-cases-optional)
+- [Requirements](#requirements)
+- [Proposal](#proposal)
```

Comment:
I see `Implementation Notes` and `Example` are part of `Proposal` so can you add those sections here